nsqd/nsqd.go

```diff
@@ -32,6 +32,12 @@ const (
 	TLSRequired
 )
 
+const (
+	NotifyTypeRegistration = iota
+	NotifyTypeUnRegistration
+	NotifyTypeStateUpdate
+)
```

Comment:
same for these const definitions, probably belong in `nsqd/lookup.go`

Reply:
done.